client/components/loading-placeholder/index.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import noop from 'lodash/utility/noop';
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import Main from 'components/main';
+
+const LoadingPlaceholder = React.createClass( {
```

Comment:
I'm not sure `LoadingPlaceholder` is the best name for this. It is actually a whole screen with a headercake and a back button. Maybe `PlaceholderScreen` or `PlaceholderPage`?